README.md

```diff
@@ -32,37 +31,36 @@ years as an in-house engine, used to publish several work-for-hire titles.
 
 #### Binary downloads
 
-Official binaries for the Godot editor and the export templates can be found
+Official binaries for the Godot editor and export templates can be found
 [on the homepage](https://godotengine.org/download).
 
-#### Compiling from source
+#### Compiling Godot from source
 
 [See the official docs](http://docs.godotengine.org/en/latest/development/compiling/)
 for compilation instructions for every supported platform.
 
 ### Community
 
-Godot is not only an engine but an ever-growing community of users and engine
+Godot is not only an engine, but an ever-growing community of users and engine
 developers. The main community channels are listed [on the homepage](https://godotengine.org/community).
 
-To get in touch with the developers, the best way is to join the
+The best way to get in touch with the Godot developers is to join the
 [#godotengine IRC channel](https://webchat.freenode.net/?channels=godotengine)
 on Freenode.
 
 ### Documentation and demos
 
 The official documentation is hosted on [ReadTheDocs](http://docs.godotengine.org).
-It is maintained by the Godot community in its own [GitHub repository](https://github.com/godotengine/godot-docs).
+It is maintained by the Godot community on its own [GitHub repository](https://github.com/godotengine/godot-docs).
```

Comment:
Here you change "in" for "on", but not in the next sentence about the demos repo.

Reply:
Great catch, thank you!